src/cpp/src/visual_language/pipeline.cpp

```diff
@@ -362,10 +373,10 @@ VLMPipeline::VLMPipeline(
         m_pimpl = std::make_unique<VLMPipelineImpl>(models_dir, device, properties);
     } else {
         // If CB is invoked explicitly, create CB adapter as is and re-throw in case if internal issues
-        if (utils::explicitly_requires_paged_attention(properties)) {
+        if (utils::explicitly_requires_paged_attention(user_properties)) {
             auto [plugin_properties, scheduler_config] = utils::extract_scheduler_config(properties, utils::get_latency_oriented_scheduler_config());
             m_pimpl = std::make_unique<VLMContinuousBatchingAdapter>(models_dir, scheduler_config, device, plugin_properties);
-        } else if (attention_backend == PA_BACKEND) {
+        } else if (attention_backend == PA_BACKEND && !requires_sdpa(models_dir)) {
```

Comment:
benmark_vlm blindly passes SchedulerConfig which is going to be ignored for qwen. https://github.com/openvinotoolkit/openvino.genai/blob/0a4dbc0a4a412223e9489f91251f3f82391aa4c2/samples/cpp/visual_language_chat/benchmark_vlm.cpp#L76
NPU gives an error (see above). I don't think we want to do the same for specific models handling them in benmark_vlm as well. But maybe we should...

I think the solution is to wrap `std::make_unique<ov::genai::VLMPipeline>(models_path, device, ov::genai::scheduler_config(scheduler_config))` with try except in the sample. Do you have other ideas?

Reply:
@sbalandi, do you know if the tools pass scheduler_config anywhere by default?

Reply:
Yeah, I mixed it with Gemma. Can you add a comment in benchmark_vlm explaining that it's going to trigger PA which is known to have accuracy issue and which is not the default bakend for these models

Reply:
Yes, done.